<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QLabel" name="gameBoard_0_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_0_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_1_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_2_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_3_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_4_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_5_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_6_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameBoard_7_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNewgame">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>NEW GAME</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoad">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoad_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>ABOUT</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="countBlack">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="BlackNum">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="countWhite">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="WhiteNum">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>261</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome! </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoad_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>50</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 170, 255);
alternate-background-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>UNDO</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>330</width>
     <height>330</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelInfo</zorder>
  <zorder>label</zorder>
  <zorder>gameBoard_0_0</zorder>
  <zorder>gameBoard_0_1</zorder>
  <zorder>gameBoard_0_2</zorder>
  <zorder>gameBoard_0_3</zorder>
  <zorder>gameBoard_0_4</zorder>
  <zorder>gameBoard_0_5</zorder>
  <zorder>gameBoard_0_6</zorder>
  <zorder>gameBoard_0_7</zorder>
  <zorder>gameBoard_1_0</zorder>
  <zorder>gameBoard_1_1</zorder>
  <zorder>gameBoard_1_2</zorder>
  <zorder>gameBoard_1_3</zorder>
  <zorder>gameBoard_1_4</zorder>
  <zorder>gameBoard_1_5</zorder>
  <zorder>gameBoard_1_6</zorder>
  <zorder>gameBoard_1_7</zorder>
  <zorder>gameBoard_2_0</zorder>
  <zorder>gameBoard_2_1</zorder>
  <zorder>gameBoard_2_2</zorder>
  <zorder>gameBoard_2_3</zorder>
  <zorder>gameBoard_2_4</zorder>
  <zorder>gameBoard_2_5</zorder>
  <zorder>gameBoard_2_6</zorder>
  <zorder>gameBoard_2_7</zorder>
  <zorder>gameBoard_3_0</zorder>
  <zorder>gameBoard_3_1</zorder>
  <zorder>gameBoard_3_2</zorder>
  <zorder>gameBoard_3_3</zorder>
  <zorder>gameBoard_3_4</zorder>
  <zorder>gameBoard_3_5</zorder>
  <zorder>gameBoard_3_6</zorder>
  <zorder>gameBoard_3_7</zorder>
  <zorder>gameBoard_4_0</zorder>
  <zorder>gameBoard_4_1</zorder>
  <zorder>gameBoard_4_2</zorder>
  <zorder>gameBoard_4_3</zorder>
  <zorder>gameBoard_4_4</zorder>
  <zorder>gameBoard_4_5</zorder>
  <zorder>gameBoard_4_6</zorder>
  <zorder>gameBoard_4_7</zorder>
  <zorder>gameBoard_5_0</zorder>
  <zorder>gameBoard_5_1</zorder>
  <zorder>gameBoard_5_2</zorder>
  <zorder>gameBoard_5_3</zorder>
  <zorder>gameBoard_5_4</zorder>
  <zorder>gameBoard_5_5</zorder>
  <zorder>gameBoard_5_6</zorder>
  <zorder>gameBoard_5_7</zorder>
  <zorder>gameBoard_6_0</zorder>
  <zorder>gameBoard_7_0</zorder>
  <zorder>gameBoard_6_1</zorder>
  <zorder>gameBoard_6_2</zorder>
  <zorder>gameBoard_6_3</zorder>
  <zorder>gameBoard_6_4</zorder>
  <zorder>gameBoard_6_5</zorder>
  <zorder>gameBoard_6_6</zorder>
  <zorder>gameBoard_6_7</zorder>
  <zorder>gameBoard_7_1</zorder>
  <zorder>gameBoard_7_2</zorder>
  <zorder>gameBoard_7_3</zorder>
  <zorder>gameBoard_7_4</zorder>
  <zorder>gameBoard_7_5</zorder>
  <zorder>gameBoard_7_6</zorder>
  <zorder>gameBoard_7_7</zorder>
  <zorder>pushButtonNewgame</zorder>
  <zorder>pushButtonSave</zorder>
  <zorder>pushButtonLoad</zorder>
  <zorder>pushButtonLoad_2</zorder>
  <zorder>countBlack</zorder>
  <zorder>BlackNum</zorder>
  <zorder>countWhite</zorder>
  <zorder>WhiteNum</zorder>
  <zorder>pushButtonLoad_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
